src/types/libdef/npm/memoize-immutable_vx.x.x.js

```diff
@@ -0,0 +1,52 @@
+// flow-typed signature: 6ea9f766ab617f8c6b05d99b0402a38b
```

Comment:
That's funny, we already have a custom version of this one in https://github.com/devtools-html/perf.html/tree/master/flow-typed/npm-custom
I think it got landed in that directory from of a PR that landed after the move :-)